_test/shared/matlab_xunit_ISISextras/@TestCaseWithSave/TestCaseWithSave.m

```diff
@@ -233,6 +233,9 @@
         
         % List of any paths to remove after test suite is completed
         paths_to_remove_= {};
+        
+        % 
+        print_running_tests_ = false;
```

Comment:
Should this be declared be in the base `TestCase` class where it's used rather than here? 

That or move the decoration print statements into an overriding implementation
```
function self = TestCase(testMethod)
  tic()
  print "start <methodname>"
 
  super.TestCase(testMethod)
  toc()
  print "done <methodname> in <time>"
```

Reply:
no -- that's leftower. Removed. The decorator is defined in TestSuite and the property itself -- in  TestComponent. To avoid multiple constructors, TestSuite generate its own TestCase for every method.